contents/articles/2017-08_pupil_mobile/index.md

```diff
@@ -0,0 +1,23 @@
+--- 
+title: Introducing Pupil Mobile
+date: Thu Aug 05 2017 14:36:00 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Pupil Mobile is a companion app to Pupil Capture and Pupil Service. The application enables you to connect your Pupil eye tracking headset to your Android device via USBC. Preview video and other sensor streams and stream video data over a WiFi..."
```

Comment:
Stream video to other clients over WiFi or record video locally on the device...